CMakeLists.txt

```diff
@@ -153,7 +153,14 @@ if (ENABLE_SCRIPTING)
 endif ()
 
 if (NOT DISABLE_DISCORD_RPC)
-    if(EXISTS "${ROOT_DIR}/discord-rpc")
+    if (UNIX AND NOT APPLE)
+        find_package(DiscordRPC)
+        if(${DISCORDRPC_FOUND})
+            set(HAVE_DISCORD_RPC ON)
+            add_definitions(-D__ENABLE_DISCORD__)
+            include_directories(DISCORDRPC_PROCESS_INCLUDES)
+        endif()
+    elseif(EXISTS "${ROOT_DIR}/discord-rpc")
```

Comment:
This will only get check in case the if in line 156, which currently reads: `if (UNIX AND NOT APPLE)` fails, meaning all linux builds would require to have the package installed system-wide.

Reply:
Should be fixed now.